librz/util/debruijn.c

```diff
@@ -7,7 +7,6 @@
 // char* peda_charset =
 //    "A%sB$nC-(D;)Ea0Fb1Gc2Hd3Ie4Jf5Kg6Lh7Mi8Nj9OkPlQmRnSoTpUqVrWsXtYuZvwxyz";
 
-//TODO(crowell): Make charset configurable, to allow banning characters.
```

Comment:
I think this TODO should stay, it's really useful feature.

Reply:
If it's useful then It should be moved to an issue, not lost in-line TODO

Reply:
i agree with Maijin. that's a valid point.